plan/analyze_test.go

```diff
@@ -42,10 +42,18 @@ func (s *testAnalyzeSuite) TestAnalyze(c *C) {
 	testKit.MustExec("create index b on t (b)")
 	testKit.MustExec("insert into t (a,b) values (1,1),(1,2),(1,3),(1,4),(2,5),(2,6),(2,7),(2,8)")
 	testKit.MustExec("analyze table t")
+
 	testKit.MustExec("create table t1 (a int, b int)")
 	testKit.MustExec("create index a on t1 (a)")
 	testKit.MustExec("create index b on t1 (b)")
 	testKit.MustExec("insert into t1 (a,b) values (1,1),(1,2),(1,3),(1,4),(2,5),(2,6),(2,7),(2,8)")
+
+	testKit.MustExec("create table t2 (a int, b int)")
+	testKit.MustExec("create index a on t2 (a)")
+	testKit.MustExec("create index b on t2 (b)")
+	testKit.MustExec("insert into t2 (a,b) values (1,1),(1,2),(1,3),(1,4),(2,5),(2,6),(2,7),(2,8)")
+	testKit.MustExec("analyze table t2 index a,b")
```

Comment:
You can add a case that create two indices but analyze only one. Then check the explain result of some statements.